packages/client-adapters/src/adapters/chineseloc/adapter.js

```diff
@@ -38,6 +38,8 @@ class AlpheiosChineseLocAdapter extends BaseAdapter {
     this._messagingService = MessagingService.getService(msgServiceName)
   }
 
+  get language () { return ChineseLanguageModel.language }
+
   get languageID () { return ChineseLanguageModel.languageID }

```

Comment:
do we still need this method?

Reply:
Unfortunately, `Feature` and some other classes still use language ID, so I think we have to have it here. Here is one of the usages of the language ID within the Chinese local adapter: https://github.com/alpheios-project/alpheios-core/blob/dev-annotations-next/packages/client-adapters/src/adapters/chineseloc/adapter.js#L166
But I think we can get rid of it some day.